### PR TITLE
add state and selector for list

### PR DESCRIPTION
**Pull request by nikki-kiga**

### CODE REVIEW

**Review by logantscott — APPROVED** [review]

**Review by MaximusLamb — APPROVED** [review]
This is so Fricking sick dude.  I so totally approve

File: src/reducers/markdownReducer.js [hunk 1]
```diff
@@ -3,7 +3,10 @@ import {
 } from '../actions/markdownActions';
 
 export const initialState = {
-  markdown: '# Hi there'
+  markdown: '# Hi there',
```
Comment (mehtaphysical) [inline src/reducers/markdownReducer.js]: why do you need both markdown and files state?

**Review by mehtaphysical — CHANGES_REQUESTED** [review]
you may have unneeded state

File: src/actions/markdownActions.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 export const UPDATE_MARKDOWN = 'UPDATE_MARKDOWN';
-export const updateMarkdown = (text) => ({
-  type: UPDATE_MARKDOWN,
-  payload: text
-});
+export const updateMarkdown = (id, title, body) => {
+  const newMarkdown = {
+    type: UPDATE_MARKDOWN,
+    payload: {}
```
Comment (mehtaphysical) [inline src/actions/markdownActions.js]:
```js
payload: {
  [id]: { title, body }
}
```

**Review by mehtaphysical — APPROVED** [review]

**Pull request merged**